uaclient/update_info.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Dict
```

Comment:
I like that you moved this subcommand code into its own file +1

Should we also create a "subcommands/" directory and move this file there? Or do you think that is too much right now?